cdap-watchdog/src/main/java/co/cask/cdap/logging/gateway/handlers/LogDataOffsetCallback.java

```diff
@@ -50,4 +57,33 @@ public Object encodeSend(LogEvent logEvent) {
                                   ThrowableProxyUtil.asString(event.getThrowableProxy()));
     return new FormattedLogDataEvent(logData, logEvent.getOffset());
   }
+
+  @Override
+  protected Object modifyLogJsonElememnt(JsonElement jsonElement) {
+    JsonObject jsonLogData = (JsonObject) jsonElement;
+    JsonObject logData = jsonLogData.getAsJsonObject("log");
+
+    if (fieldsToSuppress != null && !fieldsToSuppress.isEmpty()) {
```

Comment:
`fieldsToSuppress` cannot be null, right? The constructor does not say it is `@Nullable`, so I think we can remove the null check.